app/uk/gov/hmrc/gform/summary/SummaryRenderingService.scala

```diff
@@ -185,14 +193,22 @@ class SummaryRenderingService(
     doc.html.replace("£", "&pound;")
   }
 
-  private def addDataToPrintPdfHTML(html: String, cache: AuthCacheWithForm)(
+  private def addDataToPrintPdfHTML(
```

Comment:
Why to have this method, when it does exactly same thing as `addDataToNotificationPdfHTML`?